starter/multiwoz/restaurant/schema.tt

```diff
@@ -93,11 +93,13 @@ class @uk.ac.cam.multiwoz.Restaurant
                           property=["# food", "# cuisine"],
                           adjective=["#"],
                           passive_verb=["serving #", "serving # food", "serving # cuisine"],
-                          verb=["serves # cuisine", "serves # food", "offer # cuisine", "offer # food", "serves", "offers"]
+                          verb=["serves # cuisine", "serves # food", "offer # cuisine", "offer # food", "serves", "offers"],
+                          base_projection=["cuisine", "type of cuisine", "food", "type of food"],
+                          verb_projection=["serve", "offer"]
                         }],
                         out price_range : Enum(cheap, moderate, expensive)
                         #[conflict_filter=['id']]
-                        #_[question=["how many stars does it have"]]
+                        #_[question=["what's the price range of the restaurant"]]
```

Comment:
that's a templatized question right? not sure why we had the stars question before but i suspect it has to do with multiwoz workers being weird rather than an error, but who knows

Reply:
I replaced it with "how expensive is the restaurant?" - something you asked for but I can generate by annotation